src/main/java/org/bahmni/sms/SMSSender.java

```diff
@@ -1,5 +1,7 @@
 package org.bahmni.sms;
 
+import org.springframework.http.ResponseEntity;
+
 public interface SMSSender {
-    String send(String phoneNumber, String message);
+    ResponseEntity send(String phoneNumber, String message);
```

Comment:
Why not send back a map or something? let the controller deal with the response entity. bubble any exception, let controller catch the exception and send a 500 error